python_modules/dagster/dagster/cli/dynamic_loader.py

```diff
@@ -208,25 +245,62 @@ def create_pipeline_loading_mode_data(info):
         raise InvalidPipelineLoadingComboError()
 
 
-DynamicObject = namedtuple('DynamicObject', 'module fn module_name fn_name object')
+class DynamicObject:
+    def __init__(self, module, module_name, fn_name):
+        self.module = module
+        self.module_name = module_name
+        self.fn_name = fn_name
+        self.object = None
+        self.coerce_to_repo = False
```

Comment:
Just eliminate the coerce_to_repo variable

Reply:
Oh I thought it was unused! Just leave it in a working state and merge. Definitely not the best use of your time!